crates/cxx-qt-gen/src/naming/cpp.rs

```diff
@@ -87,14 +91,14 @@ pub(crate) fn syn_type_to_cpp_type(ty: &Type, type_names: &TypeNames) -> Result<
                 if let Lit::Int(len) = &len.lit {
                     len.base10_parse::<usize>()?
                 } else {
-                    return Err(Error::new(ty.span(), "Array length must be a integer"));
+                    return Err(Error::new(ty.span(), "Array length must be an integer!"));
```

Comment:
Hm, code duplication with line 97. :thinking: 

Can we maybe merge the two consecutive `if let` expressions into one somehow?

Something like:
```rust
if let Expr::Lit(ExprLit { lit: Lit::Int(len), ...}) = &len
```

Reply:
Hm, you should be able to get rid of the `attrs: _empty` part with `..` (two dots only, not three, sorry)
See also: https://doc.rust-lang.org/rust-by-example/flow_control/match/destructuring/destructure_structures.html

Overall I think that's cleaner, as you're destructuring in one go.